cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -483,6 +483,37 @@ def _add_imagebuilder_components(self, build_tags, lambda_cleanup_policy_stateme
         if self.config.build.components:
             self._add_custom_components(components, lambda_cleanup_policy_statements, components_resources)
 
+        validate_component_resource = imagebuilder.CfnComponent(
```

Comment:
this component is meaningful only if the main pcluster component it's executed. For this reason this new block should be encapsulated in a `if not disable_pcluster_component:`

Reply:
done